spectral_cube/conftest.py

```diff
@@ -282,6 +282,26 @@ def data_vda_beams(tmp_path):
     return tmp_path / 'vda_beams.fits'
 
 
+@pytest.fixture
+def data_vda_beams_image(tmp_path):
```

Comment:
Yes this should be sufficient and no need to then commit the .image directory

You can also make it even simpler by using one of the FITS fixtures and this: https://github.com/radio-astro-tools/spectral-cube/blob/master/spectral_cube/tests/test_casafuncs.py#L151

Reply:
I removed the `.image` directory, now we just use this.